lib/page/login/login_page.dart

```diff
@@ -49,74 +48,67 @@ class _LoginPageState extends ConsumerState<LoginPage> {
       );
     });
 
-    return Stack(
-      children: [
-        Container(
-          decoration: BoxDecoration(
-            image: DecorationImage(
-              image: AssetImage(Assets.images.bgLogin.path),
-              fit: BoxFit.cover,
-            ),
+    return Scaffold(
+      resizeToAvoidBottomInset: false,
+      body: Stack(
+        children: [
+          OnboardingBackgroundWidget(
+            background: AssetImage(Assets.images.bgOnboarding.path),
+            shouldBlur: true,
           ),
-          child: BackdropFilter(
-            filter: ImageFilter.blur(
-              sigmaX: 30,
-              sigmaY: 30,
-            ),
-            child: Container(
-              padding: const EdgeInsets.symmetric(horizontal: Dimens.space24),
-              decoration: BoxDecoration(color: Colors.black.withOpacity(0.4)),
-              child: SafeArea(
-                child: Column(
-                  children: [
-                    Container(
-                      margin: const EdgeInsets.only(top: Dimens.space120),
-                      child: Assets.images.icNimble.svg(),
-                    ),
-                    Container(
-                      margin: const EdgeInsets.only(top: Dimens.space110),
-                      child: TextInputWidget(
-                        hintText: AppLocalizations.of(context)!.loginEmail,
-                        controller: _emailController,
-                      ),
-                    ),
-                    Container(
-                      margin: const EdgeInsets.only(top: Dimens.space20),
-                      child: TextInputWidget(
-                        hintText: AppLocalizations.of(context)!.loginPassword,
-                        isPasswordInput: true,
-                        controller: _passwordController,
-                        endWidget: LoginTextInputForgotPasswordWidget(
-                          onPressed: () {
-                            _navigateToResetPassword();
-                          },
-                        ),
-                      ),
+          Container(
+            padding: const EdgeInsets.symmetric(horizontal: Dimens.space24),
+            decoration: BoxDecoration(color: Colors.black.withOpacity(0.4)),
```

Comment:
Should this effect belong to the `OnboardingBackgroundWidget` instead? I see you're trying to create a gray background layer between the background image + the screen content in SafeArea for both screens 💭

Reply:
Good point, this is fixed in #88 🙏🏼